.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: 🔍 Dependency Review
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+
+permissions: {}
+
+jobs:
+  dependency-review:
+    name: Dependency Review
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Dependency Review
+        id: dependency_review
+        uses: actions/dependency-review-action@3b139cfc5fae8b618d3eae3675e383bb1769c019 # v4.5.0
+        with:
+          fail-on-severity: critical
+ 
```

Comment:
Can this be updated? looks odd on the GitHub UI

Reply:
<img width="300" alt="Screenshot 2024-12-06 at 16 44 46" src="https://github.com/user-attachments/assets/d0438ce7-b682-4cf8-906f-70577e35710a">